src/loose/statement.js

```diff
@@ -375,7 +375,7 @@ lp.parseImportSpecifierList = function() {
   if (this.tok.type === tt.star) {
     let elt = this.startNode()
     this.next()
-    if (this.eatContextual("as")) elt.local = this.parseIdent()
+    elt.local = this.eatContextual("as") ? this.parseIdent() : this.dummyIdent();
```

Comment:
Sure. I am trying to see what I need to do to reset my fork to remote master and then reapply the same fix without the semicolon.